x/ibc/09-localhost/header.go

```diff
@@ -0,0 +1,35 @@
+package localhost
+
+import (
+	clientexported "github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment/types"
+)
+
+var _ clientexported.Header = Header{}
+
+// Header defines the Localhost consensus Header
+type Header struct {
+}
+
+// ClientType defines that the Header is in loop-back mode
+func (h Header) ClientType() clientexported.ClientType {
+	return clientexported.Localhost
+}
+
+// ConsensusState returns the consensus state associated with the header
+func (h Header) ConsensusState() ConsensusState {
+	return ConsensusState{
+		Root: commitment.NewMerkleRoot([]byte{}),
+	}
```

Comment:
I think we can return an empty consensus state
```suggestion
	return ConsensusState{}
```